app/boards/arm/dao/Kconfig.defconfig

```
@@ -0,0 +1,52 @@
+# Copyright (c) 2023 ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if BOARD_DAO_LEFT
+
+	config ZMK_KEYBOARD_NAME
+	default "Dao"
+
+    config ZMK_SPLIT_BLE_ROLE_CENTRAL
```

```suggestion
    config ZMK_SPLIT_ROLE_CENTRAL
```
The Kconfig name was updated some time ago -- also you set this in `dao_left_defconfig` as well, you could remove it from one of these places.

---

Fixed